src/ThingsBoard.h

```diff
@@ -31,15 +31,6 @@
 #include "Provision_Callback.h"
 #include "OTA_Update_Callback.h"
 
-#if !THINGSBOARD_ENABLE_PROGMEM
```

Comment:
Unfortunately the library doesn't works on chips based on RP2040 without these definitions, it cannot find methods snprinf_P and vsnprintf_P.

Reply:
@imbeacon That is correct but the defines should be included via. the definitions in the Constant.h file included above.

Reply:
Ohh, I see.